editor.planx.uk/src/@planx/components/shared/Radio/BasicRadio.tsx

```diff
@@ -15,6 +15,7 @@ const BasicRadio: React.FC<Props> = ({ id, onChange, title }) => (
     onChange={onChange}
     control={<Radio />}
     label={title}
+    sx={{ pb: 1 }}
```

Comment:
This leads to all instances of `BasicRadio` (including the final one) having additional padding-bottom. Would it be better to control this at the parent container to avoid the double spacing above the "Continue" button?

Reply:
I had considered omitting the final spacing, but the extra 10px at the bottom of the component gives good vertical balance, when compared to the description and image version which have borders + margins on each element.

Reply:
Totally fair - it looks better to me - just wanted to check it was intentional!